10LessonTen/Checkpoint2/index.html

```diff
@@ -1,10 +1,108 @@
+    <link rel="stylesheet" href="css/css-reset.css">
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
+    <header>
+        <nav>
+            <a class="navlinkmain"> NEWPORT RI, MANSIONS </a>
+            <a class="nav-link" href="#home">Home</a>
+            <a class="nav-link" href="#about">About</a>
+            <a class="nav-link" href="#explore">Explore</a>
+            <a class="nav-link" href="#shop">Shop</a>
+            <a class="nav-link" href="#social">Social</a>
+        </nav>
+    </header>
+    <main>
+        <section class="herospace">
+            <h1> ALL HOUSES OPEN DAILY </h1>
+            <h2> <p> Plan your visit </h2> </p>
+            <button> <a href="http://www.newportmansions.org/plan-a-visit"> Learn more > </button> </a>
+        </section>
+        <section class="clearfix" id="about">
+            <div class="bellevue">
+                <img src="http://www.newportmansions.org/images/content-pages/about%20us/424-bellevue.jpg" alt="BellevueMansions">
+            </div>
+
+            <div class="aboutus">
+                <h3>About Us</h3>
+                <p>
+                    The Preservation Society of Newport County, Rhode Island's largest cultural organization, protects, preserves and proesents the best of Newport County's architectural heritage. Its 11 historic properties and landscapes - seven of which are National Historic
+                    Landmarks - trace America's architectural and social development from the Colonial era through the Gilded Age. In keeping with its mission, the Society strives to offer its members and the public a comprehensive view of each property's
+                    architecture, interiors, landscapes and social history.
+                </p>
+                <p>
+                    <br> The Society is headquartered in a three-story Romanesque Revival mansion at 424 Bellevue Avenue, constructed in 1888 as a summer residence for William H. Osgood of New York. Its last private owner was Herbert Claiborne Pell, former
+                    U.S. Ambassador to Portugal and Hungary. After several incarnations as a school building, the property was purchased in 1992 by the Preservation Society, which conducted an extensive restoration and renovation of the building for use
+                    as its administrative headquarters.
+                </p>
+            </div>
+        </section>
+
+        <section class="clearfix" id="explore">
+            <div class="left1">
+                <img src="http://www.newportmansions.org/images/sections/Breakers-music-room-2.jpg" alt="TheBreakers">
+                <h4> The Breakers </h4>
+                <p>The Breakers is the grandest of Newport's summer "cottages" and a symbol of the Vanderbilt family's social and financial preeminence in turn of the century America.</p>
+                <button> <a href="http://www.newportmansions.org/explore/the-breakers" target="_blank"> Read more... </button> </a>
```

Comment:
`<button><a>...</a></button>` it's probably not a good combination.
You can make a `<a>...</a>` look like a button by using CSS instead